### PR TITLE
armstrong number added

### PR DESCRIPTION
**Pull request by tk315**
## Related Issue
-Project to find whether the entered number is Armstrong or not
-Armstrong no: number that is equal to the sum of cubes of its digits

Closes: #731 

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
armstrong_no

### CODE REVIEW

**Comment by 991rajat** [comment]
@tk315  Please add the project to script.js.

File: script.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 let projectData = [
```
Comment (991rajat) [inline script.js:1]: Please don't change the existing styling.
Are you using any kind of formatter, if using disable it while saving this file.
It is changing in every line of the project.
Only required changes need to be reflected and yours is changing everything.
fix it.
Also squash the commits, if possible if you don't know
comment here.

Comment (tk315) [reply script.js]: Yes, earlier i was using formatter i.e why i made this much commits 
Now I have corrected it.

**Comment by 991rajat** [comment]
Can you squash the commits please?

**Comment by tk315** [comment]
I am not able to stash
Capture
When i am  stashing it is showing you have files in non-stage area
I have made no change in these 3 files then also they are coming in non staging  area and are not getting removed.
I even tried cloning remote again

**Comment by 991rajat** [comment]
Stashing is totally different thing. It's not required 
Squashing is different. It combines multiple commits to one.
Don't worry about squashing for now, Ill fix that 
Right now 
CSS is not working 
Number check is not working

**Comment by tk315** [comment]
Now everything is fine 
Kindly review @991rajat

**Review by 991rajat — APPROVED** [review]
Good Work.

**Comment by 991rajat** [comment]
Congratulations on merging of first PR. :tada: 
Keep Contributing.

**Pull request merged**